package.json

```diff
@@ -2,7 +2,7 @@
   "name": "react-storefront",
   "version": "1.0.0",
   "description": "",
-  "main": "index.js",
+  "main": "server.js",
```

Comment:
I'm leaving this comment here, but is not closely related to this line.

Was there a reason for having three different files named index.js? I was sometimes confused by the naming.

Reply:
Good point. There isn't a significant reason, but I agree that they can be renamed to something more explicit.